package.json

```diff
@@ -70,6 +70,7 @@
   },
   "dependencies": {
     "ip6addr": "^0.2.5",
-    "maxmind": "^4.2.0"
+    "maxmind": "^4.2.0",
+    "rimraf": "^6.0.1"
```

Comment:
https://github.com/maxmind/GeoIP2-node/actions/runs/10818012537/job/30012693634?pr=1281#step:6:9. Added because of this `npm run build` error: https://github.com/maxmind/GeoIP2-node/actions/runs/10818012537/job/30012693634?pr=1281#step:6:9

Reply:
Could this be a dev dependency? I don't think we need it at runtime.

Reply:
Good call!

Reply:
You could also replace `rimraf` with a simple call to `rm -rf` since we don't expect to run these build scripts on windows.  If non-maxmind contributors are on windows, we can revisit.

https://github.com/maxmind/GeoIP2-node/blob/9a42e80150b820ea786f58cf1ae9ad66ab71eae5/package.json#L59-L60

Replace with:
```
"build": "rm -rf dist && tsc",
"build:docs": "rm -rf docs && typedoc src/index.ts --out docs --exclude \"**/*.spec.ts\" --readme README.md && ./bin/sanitize-doc-refs.sh",
```